source/constructs/api/catalog/service.py

```diff
@@ -286,6 +287,7 @@ def sync_crawler_result(
             tables_response = glue_client.get_tables(
                 DatabaseName=glue_database_name, NextToken=next_token
             )
+            print(f"tableList is {tables_response['TableList']}")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you have passed parameters to the `print` method that can expose sensitive information in console. This can compromise user privacy and is often illegal.

[Learn more](https://cwe.mitre.org/data/definitions/200.html)

Reply:
resloved.